woocommerce/woocommerce
# Build release workflow

### All Submissions:






### Changes proposed in this Pull Request:



Introduces a build workflow to generate a zip file and save it in the GitHub release page.

### How to test the changes in this Pull Request:

1. Only possible to test after merged on master, so fork WooCommerce and merge `add/build-workflow` to `master`.
2. On your fork you can now create a release, with any release number you like, it will fetch all code from the tag or branch you selected and build everything.
3. After saved the release you can watch everything in the `/actions` page, it generally takes 4 minutes to complete, after that you can check for the zip file in the release page. 

### Other information:
